src/components/CourseCard/index.tsx

```diff
@@ -0,0 +1,42 @@
+import Typography from '@mui/material/Typography';
+import CardContent from '@mui/material/CardContent';
+import Card from '@mui/material/Card';
+import Link from 'next/link';
+import { CourseWithTagsAndStudentCount } from '@/lib/prisma/courses';
+import { getCourseDateString } from '@/lib/util';
+
+type Props = {
+  course: CourseWithTagsAndStudentCount;
+};
+
+const CourseCard = ({ course }: Props) => {
+  const courseDate = getCourseDateString(course);
```

Comment:
Maybe this could be done in a separate task after that branch has been merged?

Reply:
Yes, It's better to leave it for after this branch has been merged to staging. I can apply the util function to the date in my branch before merging mine to staging, so you don't have to worry about it. We can adjust later the formatting function for best possible user experice. It will need some tweaks anyway after i18n branch is merged, as we are using two locales (fi/en). I was thinking we just pass the locale as a parameter later to the function and it gives the desired formatting.

I'll go to through this PR fast tomorrow morning and see if there is something. Been in extreme rush past few days.